Makefile

```diff
@@ -35,7 +37,7 @@ tests-once: install-dev
 
 tests:
 	@rm -fr .coverage
-	tox
+	$(VENV)/bin/tox
```

Comment:
the Make is supposed to work on a fresh python install, and you are installing tox locally in your makefile, so why not use the tox you installed ?